build/__finalStretchVersions.sh

```diff
@@ -0,0 +1,8 @@
+# This file was auto-generated from 'constants.yaml'. Changes may be overridden.
+
+FINAL_STRETCH_DOT_NET_CORE_31_SDK_VERSION='3.1.420'
```

Comment:
Any reason we only need to consider the final stretch version for .NET and no other platforms? Is it because we aren't pre-baking any version of other platforms that are currently supported anymore? (_e.g._, Python 3.7 and 3.8 in the `lts-versions` build image)

Reply:
Oh yeah some much needed context is that there is [a ticket](https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1599477) coming down the line which @waliMSFT will be working on to do this for any other versions / platforms that are needed.

This PR is just a stopgap until then since if we currently use the prod storage account to pre-bake images it will break, and my parameterization pull request switches Oryx CI to use prod to pre-bake.

Once this PR is merged, we should be free to remove the dotnet 3.1.422 and 6.0.400 stretch versions from our dev storage account, reconciling it with what exists in our versionsToBuild.txt file

Reply:
Cool, sounds good :shipit: